src/App.vue

```diff
@@ -44,9 +44,9 @@ export default {
 
     this.$store.dispatch('setState', state)
     this.$store.dispatch('checkIfOnline')
-    if (window.web3) {
-      this.$store.dispatch('setWeb3Instance', window.web3)
-    }
+    // if (window.web3) {
```

Comment:
This is checking for any instance of web3 in the window. This should be kept. Along with the line 23 change.

Reply:
![image](https://user-images.githubusercontent.com/12111403/43874546-f2c2dc56-9bbe-11e8-8857-ecdf27d6c786.png)

I try to print web3's version , result is 0.20.3 . ,   but the dependent version is  1.0.0-beta.34  , what is the reason?